client/src/components/SupermarketProductsInfo.js

```diff
@@ -0,0 +1,38 @@
+import React from "react";
+import PropTypes from "prop-types";
+import styled from "@emotion/styled";
+
+function SupermarketProductsInfo({ supermarket }) {
+  return (
+    <Container>
+      <p>
+        Products found:{" "}
```

Comment:
See comment above

Reply:
Yes. otherwise it will not add whitespace here